Sources/MapboxMaps/Style/Style.swift

```diff
@@ -281,13 +281,12 @@ public final class Style: StyleProtocol {
     /// Get or set the map style's transition options.
     ///
     /// By default, the style parser will attempt to read the style default
-    /// transition options, if any, falling back to an immediate transition
-    /// otherwise.
+    /// transition, if any, falling back to an immediate transition otherwise.
```

Comment:
I think the style's default transition is not immediate, but transition with 300ms duration https://docs.mapbox.com/mapbox-gl-js/style-spec/transition/#duration